### PR TITLE
[Backup] `az backup vault encryption update`: Add support for MUA for CMK operations

### PR DESCRIPTION
**Pull request by zubairabid**
**Related command**

`az backup vault encryption update`

**Description**

Multi-User Authorization (MUA) now works for update encryption tasks in recovery services vaults.

**Testing Guide**

**History Notes**

[backup] `az backup vault encryption update`: Add support for MUA for CMK operations

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
backup

**Comment by zubairabid** [comment]
> Hi @zubairabid, Since the current milestone time is less than 7 days, this pr will be reviewed in the next milestone.

Requesting a review within this milestone.

**Comment by yanzhudd** [comment]
please fix the CI issues.

**Comment by yanzhudd** [comment]
Please add test for this change to verify if it meets your expectations.

**Comment by zubairabid** [comment]
> Please add test for this change to verify if it meets your expectations.

@yanzhudd the code has been tested manually. We are not adding an automated test at the moment as it involves cross-tenant calls being made, and the existing automated tests that verify this general functionality are alright.

Resolving the CI issues.

**Comment by zubairabid** [comment]
> > Please add test for this change to verify if it meets your expectations.
> 
> @yanzhudd the code has been tested manually. We are not adding an automated test at the moment as it involves cross-tenant calls being made, and the existing automated tests that verify this general functionality are alright.
> 
> Resolving the CI issues.

@yanzhudd I'm working on resolving the CI issues - they're related to an unrelated RBAC change and are mostly fixed. But currently my local testing is getting DOS limited so I'll be able to push the fix for it mostly tomorrow, but I've added a unittest skip for today. Can you please approve the PR (we have been targeting this month's release).

**Comment by yanzhudd** [comment]
Hi @zubairabid,
No worries. We have pushed the release back to next week. As long as these issues are resolved by 07-02, this PR will be ready to catch the upcoming release train.

File: src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py [hunk 1]
```diff
@@ -1179,6 +1179,7 @@ def test_backup_identity(self, resource_group, vault_name):
     @ResourceGroupPreparer()
     @VaultPreparer(parameter_name='vault1')
     @VaultPreparer(parameter_name='vault2')
+    # @unittest.skip('RBAC-related issue, TODO fix by June 29 2024')
```
Comment (yanzhudd) [inline src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py:1182]: It is better to remove these comments.
```suggestion
```

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**